### PR TITLE
[squid-dao] Add squid-dao strategy

### PR DESCRIPTION
**Pull request by zencephalon**
Adds voting strategy for SquidDAO, combining NFT holders and vewsSquid holders.

### CODE REVIEW

File: src/strategies/squid-dao/schema.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/Strategy",
+  "definitions": {
+    "Strategy": {
+      "title": "Strategy",
+      "type": "object",
+      "properties": {
+        "decimals": {
```
Comment (ChaituVR) [inline src/strategies/squid-dao/schema.json:9]: Thanks for adding this schema :) 
Can you also add symbol param here, for example: 
https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/erc20-balance-of/schema.json#L9-L13

File: src/strategies/squid-dao/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# squid-dao
+
+Combines SquidDAO vewsSquid holders and NFT holders, returning a combined voting power as a percentage of the total.
+
+```json
+{
+  "decimals": 4
```
Comment (ChaituVR) [inline src/strategies/squid-dao/README.md:7]:
```suggestion
  "symbol": "DAI",
  "decimals": 4
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/squid-dao/schema.json (hunk 1 above)
Comment (zencephalon) [reply src/strategies/squid-dao/schema.json:14]: Done!

File: src/strategies/squid-dao/README.md (hunk 2 above)
Comment (zencephalon) [reply src/strategies/squid-dao/README.md]: Thanks!

**Comment by zencephalon** [comment]
@ChaituVR Thanks! Please take another look now. :)

File: src/strategies/squid-dao/schema.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```
Comment (bonustrack) [inline src/strategies/squid-dao/schema.json:2]: @ChaituVR Did you tested this schema? We should test it against examples.json

Comment (zencephalon) [reply src/strategies/squid-dao/schema.json:2]: Yes, the tests pass!

```
npm run test --strategy=squid-dao                                                            09:38:44

> @snapshot-labs/strategies@0.1.0 pretest
> npm run build


> @snapshot-labs/strategies@0.1.0 build
> tsc -p .


> @snapshot-labs/strategies@0.1.0 postbuild
> copyfiles -u 1 src/**/*.md dist/ && copyfiles -u 1 src/**/*.json dist/


> @snapshot-labs/strategies@0.1.0 test
> jest test/index.spec.ts --runInBand --testTimeout=120000

  console.log
    [
      {
        '0xfd4f24676ed4588928213f37b126b53c07186f45': 5.1986,
        '0x0f763341b448bb0f02370f4037fe4a2c84c9283f': 94.8013
      }
    ]

      at Object.<anonymous> (test/index.spec.ts:47:13)

  console.log
    Resolved in 1.22 sec.

      at Object.<anonymous> (test/index.spec.ts:48:13)

  console.log
    Scores with latest snapshot [
      {
        '0xfd4f24676ed4588928213f37b126b53c07186f45': 5.1925,
        '0x0f763341b448bb0f02370f4037fe4a2c84c9283f': 94.8074
      }
    ]

      at Object.<anonymous> (test/index.spec.ts:110:13)

  console.log
    Resolved in 0.84 sec.

      at Object.<anonymous> (test/index.spec.ts:111:13)

 PASS  test/index.spec.ts (5.197 s)
  
Test strategy "squid-dao"
    ✓ Strategy should run without any errors (1236 ms)
    ✓ Should return an array of object with addresses (2 ms)
    ✓ Should take less than 10 sec. to resolve (1 ms)
    ✓ File examples.json should include at least 1 address with a positive score
    ✓ File examples.json must use a snapshot block number in the past (569 ms)
    ✓ File examples.json must have symbol in its strategy params
    ✓ Returned addresses should be either same case as input addresses or checksum addresses (1 ms)
  
Test strategy "squid-dao" with latest snapshot
    ✓ Strategy should run without any errors (1338 ms)
    ✓ Should return an array of object with addresses (1 ms)
  
Test strategy "squid-dao" (with 500 addresses)
    ○ skipped Should work with 500 addresses
    ○ skipped Should take less than 20 sec. to resolve with 500 addresses
  
Others:
    ✓ Author in strategy should be a valid github username (200 ms)
    ✓ Version in strategy should be a valid string

Test Suites: 1 passed, 1 total
Tests:       2 skipped, 11 passed, 13 total
Snapshots:   0 total
Time:        6.054 s
```

Comment (ChaituVR) [reply src/strategies/squid-dao/schema.json:2]: @bonustrack Yes validated the schema against examples.json and no errors in it :) should be good to go

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**